storyblok/storyblok-design-system
# Refactor SbCard Component to SFC

## Pull request type

Jira Link: [INT-](url)




## How to test this PR



Please navigate to the [SbCard page](http://localhost:6006/?path=/docs/design-system-components-sbcard--default) in Storybook and do the following:

1. verify that the card responds to the `as`, `title`, and `isLoading` props as expected
2. verify that the options button only appears in the card header in the section labeled "With Menu Component" and that the options menu works as expected.

## What is the new behavior?



- No new behavior, just refactored code.

## Other information
